snomed/com.b2international.snowowl.snomed.ql/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove the `.antlr-generator-3.2.0-patch.jar` from the `snomed.ql` module and use a `.gitignore` file to prevent it from being added to git later.